pkgs/by-name/cl/clipcat/package.nix

```diff
@@ -20,6 +20,8 @@ rustPlatform.buildRustPackage rec {
 
   cargoHash = "sha256-UA+NTtZ2qffUPUmvCidnTHwFzD3WOPTlxHR2e2vKwPQ=";
 
+  patches = [ ./dummy.patch ];
```

Comment:
Would be helpful to add a note explaining the purpose of the patch so that it can get cleaned up.

Reply:
Done.